rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -81,23 +81,21 @@ object Runtime {
   type Remainder = Option[Var]
   type BodyRef   = Long
 
-  // parentHash in side the BlockData is only used for closeBlock system deploy random initial
-  // parentHash is not exposed in the rholang contract `rho:block:data`
-  // maybe future we will refactor this into a better way
+  // seqNum is not exposed to rholang api contract currently
   final case class BlockData private (
       timeStamp: Long,
       blockNumber: Long,
       sender: PublicKey,
-      parentHash: Array[Byte]
```

Comment:
Was this parentHash in BlockData only being used to generate random seeds for system deploys?

Reply:
Yes, you can see in the comments. ;)